src/components/OfflineSearch.ts

```diff
@@ -17,6 +19,7 @@ export default class OfflineSearch extends Component<OfflineSearchProps, Offline
             findingWidget: true
         };
         this.updateConstraints = this.updateConstraints.bind(this);
+        dojoConnect.connect(props.mxform, "onNavigation", this, dojoLang.hitch(this, this.initSearch));
```

Comment:
When connecting to events, you should always unsubscribe to the evenr
``` ts
this.navigationHandler = dojoConnect.connect(props.mxform, "onNavigation", this, dojoLang.hitch(this, this.initSearch));
// on unmounting
dojoConnect.disconnect(this.navigationHandler);
```

Reply:
Good one

Reply:
Done